data/assets/base.asset

```diff
@@ -87,4 +87,13 @@ asset.onInitialize(function()
   openspace.globebrowsing.loadWMSServersFromFile(
     openspace.absPath("${DATA}/globebrowsing_servers.lua")
   )
+
+  openspace.setGuiTreeOrdering(
+    "/",
+    { "Solar System", "Milky Way", "Universe", "Night Sky" }
```

Comment:
```suggestion
    { "Solar System", "Milky Way", "Universe", "Night Sky", "Examples" }
```

Just noticed when doing some more examples and it would be useful

Reply:
Sure thing